Description: Disable hedge feature with failed to build dep hdrhistogram
 hdrhistogram fails to build on i386, blocking itself and tower from migrating
 to testing
Bug: https://github.com/HdrHistogram/HdrHistogram_rust/issues/111
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1014034
Last-Update: 2022-11-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -55,10 +55,10 @@
 name = "tower"
 path = "src/lib.rs"
 
-[[example]]
-name = "tower-balance"
-path = "examples/tower-balance.rs"
-required-features = ["full"]
+#[[example]]
+#name = "tower-balance"
+#path = "examples/tower-balance.rs"
+#required-features = ["full"]
 
 [[test]]
 name = "balance"
@@ -118,10 +118,10 @@
 optional = true
 default-features = false
 
-[dependencies.hdrhistogram]
-version = "7.0"
-optional = true
-default-features = false
+#[dependencies.hdrhistogram]
+#version = "7.0"
+#optional = true
+#default-features = false
 
 [dependencies.indexmap]
 version = "2.0.2"
@@ -168,9 +168,9 @@
 [dev-dependencies.futures]
 version = "0.3.22"
 
-[dev-dependencies.hdrhistogram]
-version = "7.0"
-default-features = false
+#[dev-dependencies.hdrhistogram]
+#version = "7.0"
+#default-features = false
 
 [dev-dependencies.http]
 version = "1"
@@ -249,7 +249,7 @@
     "buffer",
     "discover",
     "filter",
-    "hedge",
+#    "hedge",
     "limit",
     "load",
     "load-shed",
@@ -262,14 +262,14 @@
     "timeout",
     "util",
 ]
-hedge = [
-    "util",
-    "filter",
-    "futures-util",
-    "hdrhistogram",
-    "tokio/time",
-    "tracing",
-]
+#hedge = [
+#    "util",
+#    "filter",
+#    "futures-util",
+#    "hdrhistogram",
+#    "tokio/time",
+#    "tracing",
+#]
 limit = [
     "__common",
     "tokio/time",
